mvpa2/mappers/procrustean.py

```diff
@@ -208,3 +208,10 @@ def _train(self, source):
             d_r = np.linalg.norm(odatas[0] - res_r)/np.linalg.norm(odatas[0])
             debug('MAP_', "%s, residuals are forward: %g,"
                   " reverse: %g" % (repr(self), d_f, d_r))
+
+
+    def _compute_recon(self):
+        """For Procrustean mapper, inverse is transpose.
```

Comment:
re 1.  yes -- additional test which tests for particularly this, that whatever is returned from recon is the inverse (thus ```np.dot(_proj, _proj.T)``` and probably ```np.dot(_proj.T, _proj)``` close to identity ).  This way, if our other tests fail we would also know if that is solely due to this or not (**unit-** test)

2. ;-) unless we want to refactor it into some pure Procrustean and ObliqueProcrustean, which I don't think we should, this inversion here should be conditioned on either oblique was requested.  And then we should have tests for 1. in various scenarios